packages/frontend/src/components/AccountListSidebar/AccountListSidebar.tsx

```diff
@@ -63,6 +63,13 @@ export default function AccountListSidebar({
 
   const [{ accounts: noficationSettings }] = useAccountNotificationStore()
 
+  // Drag and drop state
+  const [draggedAccountId, setDraggedAccountId] = useState<number | null>(null)
+  const [dropIndicator, setDropIndicator] = useState<{
+    index: number
+    position: 'top' | 'bottom'
```

Comment:
Looking at behavior, it seems that `{ index: 0, position: 'bottom' }` is not much different from `{ index: 1, position: 'top' }`? But I guess it's fine.